app/dashboard/views.py

```diff
@@ -1100,6 +1099,59 @@ def bounty_details(request, ghuser='', ghrepo='', ghissue=0, stdbounties_id=None
     return TemplateResponse(request, 'bounty/details.html', params)
 
 
+def funder_payout_reminder_modal(request, bounty_network, stdbounties_id):
+    bounty = Bounty.objects.current().filter(network=bounty_network, standard_bounties_id=stdbounties_id).first()
+
+    context = {
+        'bounty': bounty,
+        'active': 'funder_payout_reminder_modal',
+        'title': _('Send Payout Reminder')
+    }
+    return TemplateResponse(request, 'funder_payout_reminder_modal.html', context)
+
+
+@csrf_exempt
+def funder_payout_reminder(request, bounty_network, stdbounties_id):
+    if not request.user.is_authenticated:
+        return JsonResponse(
+            {'error': 'You must be authenticated via github to use this feature!'},
+            status=401)
+
+    if hasattr(request.user, 'profile'):
+        access_token = request.user.profile.get_access_token()
+    else:
+        access_token = request.session.get('access_token')
+    github_user_data = get_github_user_data(access_token)
+
+    try:
+        bounty = Bounty.objects.current().filter(network=bounty_network, standard_bounties_id=stdbounties_id).first()
+    except Bounty.DoesNotExist:
+        raise Http404
+
+    has_fulfilled = bounty.fulfillments.filter(fulfiller_github_username=github_user_data['login']).count()
+    if has_fulfilled == 0:
+        return JsonResponse({
+            'success': False,
+          },
+          status=403)
+
+    #  410 Gone Indicates that the resource requested is no longer available and will not be available again.
+    if bounty.funder_last_messaged_on:
```

Comment:
Is the idea here that the funder can only be reminded once?

Reply:
@danlipert Exactly!